docs/DeveloperGuide.md

```diff
@@ -159,8 +159,14 @@ This section describes some noteworthy details on how certain features are imple
 The implementation of theme is done by parsing the raw values of the `json` file containing a color palette, and then
 transforming them into a `css` file that is then applied to `MainWindow.fxml`. All other elements will read from the
 same `css` file. When the application is first launched, it will apply the default theme constructed by `ThemeFactory`.
-When a theme is set by the command, it and it's temporary `css` file will be stored as variables in `ThemeManager`,
-which will subsequently be used by the other parts of the application (for e.g. messageboxes).
+When a theme is set by the command, it and its temporary `css` file will be stored in an instance of `ThemeManager`. It
```

Comment:
What does `it` refer to?

Reply:
![image](https://user-images.githubusercontent.com/65599808/114275222-49bd3880-9a54-11eb-8d75-56671bf825b7.png)